src/main/java/racinggame/domain/RacingGame.java

```diff
@@ -1,46 +1,20 @@
 package racinggame.domain;
 
-import java.util.List;
-import racinggame.view.GameView;
-
 public class RacingGame {
 
-    public static final String QUIT = "q";
     private static final int RANGE_OF_RANDOM_NUMBER = 9 - 0 + 1;
 
-    private GameView gameView;
-    private List<RacingCar> racingCars;
-    private int trial;
+    private RacingCars racingCars;
 
     public RacingGame() {}
 
-    public RacingGame(List<RacingCar> racingCars, int trial) {
-        this.gameView = new GameView(racingCars);
+    public RacingGame(final RacingCars racingCars) {
+        Validator.validatePossibleToStart(racingCars);
         this.racingCars = racingCars;
-        this.trial = trial;
-    }
-
-    public void start() {
-        for (int i = 0; i < trial; i++) {
-            System.out.println(String.format("계속 진행 하려면 아무키나 입력해 주세요.(종료 = %s)", QUIT));
-            String command = Utils.getInput();
-
-            if (isQuit(command)) {
-                System.exit(0);
-            }
-            progress();
-
-            gameView.printProgress();
-        }
-        gameView.printWinners();
     }
 
-    public static boolean isQuit(String command) {
-        return command.equals(QUIT);
-    }
-
-    private void progress() {
-        racingCars.forEach(car -> {
+    public void progress() {
+        racingCars.get().forEach(car -> {
             int randomValue = getRandomValue();
             if (car.canForward(randomValue)) {
                 car.forward();
```

Comment:
```suggestion
            car.forward();
```
전진 유무를 질의한 다음 전진 행위를 진행하는 것보다 자동차 전진 행위에 전진 유무 기능이 포함하면, 유연한 코드가 될 수 있을 것입니다.

Reply:
감사합니다!!! 반영하였습니다!